SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestHandoverReplication/SpatialTestHandoverActorComponentReplication.cpp

```diff
@@ -36,21 +36,39 @@ void AHandoverReplicationTestCube::SetTestValues(int UpdatedTestPropertyValue)
 {
 	HandoverTestProperty = UpdatedTestPropertyValue;
 	ReplicatedTestProperty = UpdatedTestPropertyValue;
+	HandoverTestStruct.FirstProperty = UpdatedTestPropertyValue;
+	HandoverTestStruct.SecondProperty = UpdatedTestPropertyValue * 2; // Just to get something different
```

Comment:
Is there a need for SecondProperty to have a value different from FirstProperty? I'd like to avoid doing this if possible as these `* 2` are everywhere (and God help us if UpdatedPropertyValue is ever 0, then `// Just to get something different` is somewhat misleading; `+ 1` should do a better job of this)

Reply:
Unfortunately, I rely on default-zero-initialization of UProps here, so needed *2 to work with the initial values.
Decided that I am not really reliant on it being something different, so made all of them use the same value.